### PR TITLE
XS✔ ◾ ✨ New rule - Successful Reachouts

### PR DESCRIPTION
**Pull request by bradystroud**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

My big brain
Our reachouts could be more successful, i want to document whats worked for me

### CODE REVIEW

File: rules/successful-reachouts/rule.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+![Figure: Typical developers LinkedIn inbox](developer-reachout-spam.png)
+
+<!-- endintro -->
+
+Here are some tips to make your reachouts stand out:
+
+1. Use emojis - a few emojis can make your message stand out, and feel more personal.
+2. Ask a question - most reachouts don't have a proper call to action, they simply ask you to "get in touch". Asking a question engages the candidate and makes them more likely to respond.  
+E.g. If you see someone is learning Rust, you could ask "How long have you been learning Rust? We have a few "Rustaceans" in the SSW Brisbane office 🦀"  
+Its important to ask the question early in the message, as most people will only glance at the .
```
Comment (tiagov8) [inline rules/successful-reachouts/rule.md:25]: Hey @bradystroud 

- [ ] 1. Fix from "Its" to "It's "
- [ ] 2. I think there is a word missing at the end "glance at the ."
- [ ] 3. Remove the full stops from the lists as per https://www.ssw.com.au/rules/avoid-full-stops-in-bullet-point-lists

File: rules/successful-reachouts/rule.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    url: https://www.ssw.com.au/people/brady-stroud
+created: 2024-03-1T01:55:35.941Z
+guid: 012a5f87-5cfb-4002-8d5f-4d2a8787ed6a
+---
+
+Developers receive a lot of reachouts. Sometimes they are automated and other times they are a person copying and pasting the same message to everyone.
+
+To have your reachout rise above the noise, you need to make it personal and unique.
+
+![Figure: Typical developers LinkedIn inbox](developer-reachout-spam.png)
```
Comment (tiagov8) [inline rules/successful-reachouts/rule.md:16]: @bradystroud 

- [ ] 1. Should we make this the "bad" example?
- [ ] 2. Consider making this a medium image... as it's too vertical (causing scrolling) https://ssw.com.au/rules/rule/#6-images

File: rules/successful-reachouts/rule.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+type: rule
+title: Do you know to make your Linkedln reachouts stand out?
```
Comment (tiagov8) [inline rules/successful-reachouts/rule.md:3]: @bradystroud 

- [ ] 1. Fix "LinkedIn"
- [ ] 2. I think we can remove the word "your"

**Review by tiagov8 — CHANGES_REQUESTED** [review]
@bradystroud 

- [ ] 1. Could we add https://www.ssw.com.au/rules/use-emojis/ as related?

**Review by sethdaily — APPROVED** [review]

File: rules/successful-reachouts/rule.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+type: rule
+title: Do you know to make LinkedIn reachouts stand out?
+uri: successful-reachouts
+authors:
+  - title: Brady Stroud
+    url: https://www.ssw.com.au/people/brady-stroud
+created: 2024-03-1T01:55:35.941Z
+guid: 012a5f87-5cfb-4002-8d5f-4d2a8787ed6a
+---
```
Comment (sethdaily) [inline rules/successful-reachouts/rule.md:10]:
```suggestion
related:
- do-you-know-the-main-features-of-linkedin-talent-hub
- when-to-hire-more-people
- do-you-use-linkedin-recruiter-to-help-you-find-more-candidates
---
```

File: rules/successful-reachouts/rule.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+authors:
+  - title: Brady Stroud
+    url: https://www.ssw.com.au/people/brady-stroud
+created: 2024-03-1T01:55:35.941Z
+guid: 012a5f87-5cfb-4002-8d5f-4d2a8787ed6a
+related:
+- do-you-know-the-main-features-of-linkedin-talent-hub
+- when-to-hire-more-people
+- do-you-use-linkedin-recruiter-to-help-you-find-more-candidates
+---
```
Comment (sethdaily) [inline rules/successful-reachouts/rule.md:14]:
```suggestion
related:
- do-you-know-the-main-features-of-linkedin-talent-hub
- when-to-hire-more-people
- do-you-use-linkedin-recruiter-to-help-you-find-more-candidates
created: 2024-03-1T01:55:35.941Z
guid: 012a5f87-5cfb-4002-8d5f-4d2a8787ed6a
archivedreason: null
---
```

**Review by sethdaily — APPROVED** [review]

**Comment by sethdaily** [comment]
@tiagov8 
1. pls approve your requested changes

File: rules/successful-reachouts/rule.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+type: rule
+title: Do you know to make LinkedIn reachouts stand out?
+uri: successful-reachouts
+authors:
+  - title: Brady Stroud
+    url: https://www.ssw.com.au/people/brady-stroud
+related:
+- do-you-know-the-main-features-of-linkedin-talent-hub
+- when-to-hire-more-people
+- do-you-use-linkedin-recruiter-to-help-you-find-more-candidates
```
Comment (JackDevAU) [inline rules/successful-reachouts/rule.md:11]:
```suggestion
  - do-you-know-the-main-features-of-linkedin-talent-hub
  - when-to-hire-more-people
  - do-you-use-linkedin-recruiter-to-help-you-find-more-candidates
```

File: rules/successful-reachouts/rule.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+title: Do you know to make LinkedIn reachouts stand out?
+uri: successful-reachouts
+authors:
+  - title: Brady Stroud
+    url: https://www.ssw.com.au/people/brady-stroud
+related:
+  - do-you-know-the-main-features-of-linkedin-talent-hub
+  - when-to-hire-more-people
+  - do-you-use-linkedin-recruiter-to-help-you-find-more-candidates
+created: 2024-03-1T01:55:35.941Z
```
Comment (JackDevAU) [inline rules/successful-reachouts/rule.md:12]:
```suggestion
created: 2024-03-01T01:55:35.941Z
```

Comment (JackDevAU) [reply rules/successful-reachouts/rule.md]: The issue seems to be that the date wasn't 100% valid ISO 8601 format 🙃

**Review by JackDevAU — APPROVED** [review]

**Pull request merged**